specs/commandline/verify.md

```diff
@@ -37,9 +37,11 @@ Usage:
 Flags:
   -d,  --debug                       debug mode
   -h,  --help                        help for verify
+       --oci-layout                  [Preview] verify the artifact stored in OCI image layout
```

Comment:
```suggestion
       --oci-layout                 verify artifact stored in OCI image layout
```

Reply:
The [Preview] label should be reserved in RC.4.